java/pom.xml

```diff
@@ -33,7 +33,7 @@
     <dep.junit.jupiter.version>5.10.1</dep.junit.jupiter.version>
     <dep.slf4j.version>2.0.9</dep.slf4j.version>
     <dep.guava-bom.version>33.0.0-jre</dep.guava-bom.version>
-    <dep.netty-bom.version>4.1.104.Final</dep.netty-bom.version>
+    <dep.netty-bom.version>siren-4.1.104-2</dep.netty-bom.version>
```

Comment:
```suggestion
    <dep.netty-bom.version>siren-4.1.104-1-SNAPSHOT</dep.netty-bom.version>
```

Reply:
The netty version has been updated: c9f0cc0fe0fd94ce19c45c94fcfcb0433fe9a936